pulp_container/app/tasks/sync_stages.py

```diff
@@ -62,7 +63,14 @@ def __init__(self, remote, signed_only):
 
     async def _download_manifest_data(self, manifest_url):
         downloader = self.remote.get_downloader(url=manifest_url)
-        response = await downloader.run(extra_data={"headers": V2_ACCEPT_HEADERS})
+        try:
+            response = await downloader.run(extra_data={"headers": V2_ACCEPT_HEADERS})
+        except PayloadTooLarge as e:
+            # if it failed to download the manifest, log the error and
+            # there is nothing to return
+            log.warning(e.args[0])
+            return None, None, None
```

Comment:
We are addressing another feature request with this change: https://github.com/pulp/pulp_container/issues/648. Should we treat a corrupted manifest with the same level of importance as a manifest larger than 4MB+?